emsdk

```diff
@@ -684,7 +692,7 @@ class Tool:
   def __init__(self, data):
     # Convert the dictionary representation of the tool in 'data' to members of this class for convenience.
     for key in data:
-      if isinstance(data[key], unicode):
+      if sys.version_info < (3,) and isinstance(data[key], str):
```

Comment:
This changed `unicode` to `str`, why is that? I think this will break people, see https://github.com/juj/emsdk/commit/81aaba9b4e34010564551649bc2e44c8321463b7 and https://github.com/kripken/emscripten/issues/3147 .